apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/CharSequenceToContainCreatorSamples.kt

```diff
@@ -0,0 +1,88 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.expect
+import kotlin.test.Test
+
+class CharSequenceToContainCreatorSamples {
+
+    @Test
+    fun value() {
+        expect("ABC") toContain exactly(1).value("A")
```

Comment:
only half infix :slightly_smiling_face: (I guess this does not compile or does it?)
More things to fix also further below

Reply:
fixed